### PR TITLE
Cap certificate download to 64KB

### PR DESCRIPTION
**Pull request by crienzo**

### CODE REVIEW

File: src/stir_shaken_service.c [hunk 1]
```diff
@@ -10,6 +10,12 @@ static size_t stir_shaken_curl_write_callback(void *contents, size_t size, size_
 
 	stir_shaken_clear_error(mem->ss);
 
+	if (http_req->max_response_download_size > 0 && mem->size + realsize > http_req->max_response_download_size) {
+		printf("WTF Exceeded max HTTP(s) download size");
```
Comment (andywolk) [inline src/stir_shaken_service.c:14]: Do we really need to say `WTF` here?

**Review by andywolk — CHANGES_REQUESTED** [review]

File: src/stir_shaken_service.c (hunk 1 above)
Comment (crienzo) [reply src/stir_shaken_service.c:14]: no :)

**Review by andywolk — APPROVED** [review]

**Comment by crienzo** [comment]
@andywolk this is all set

**Comment by crienzo** [comment]
@andywolk this is all set

**Comment by crienzo** [comment]
Squash when merging, please

**Pull request merged**